db/migrate/20200806052718_create_profiles.rb

```diff
@@ -0,0 +1,10 @@
+class CreateProfiles < ActiveRecord::Migration[6.0]
+  def change
+    create_table :profiles do |t|
+      t.belongs_to :user, null: false, foreign_key: { on_delete: :cascade }
```

Comment:
Under certain conditions (internal admin tools) we use `delete` instead of `destroy` on users, so apart from `dependent: destroy` on the model layer I also added cascading to the DB.